src/components/exercises/00002-rr-no1/01-atoms/TableRow/TableRow.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from './TableRow.scss';
+
+import ColorSquare from '../ColorSquare/ColorSquare';
+
+/*
+ * Single row html structure
+ */
+const TableRow = (props) => {
```

Comment:
If you ever need only one property from `props`, it's a good idea to use ES6 argument destructuring like so:

```javascript
const TableRow = ({ carDetails }) => {
```